docker/px4-dev/Dockerfile_simulation

```diff
@@ -14,14 +14,19 @@ RUN wget --quiet http://packages.osrfoundation.org/gazebo.key -O - | apt-key add
 		gstreamer1.0-plugins-base \
 		libeigen3-dev \
 		libgazebo9-dev \
+		libgtest-dev \
 		libimage-exiftool-perl \
 		libopencv-dev \
 		libxml2-utils \
 		pkg-config \
 		protobuf-compiler \
 	&& apt-get -y autoremove \
 	&& apt-get clean autoclean \
-	&& rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
+	&& rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/* \
+	# gtest
```

Comment:
Can you put this on a separate `RUN`?

Reply:
Yes of course. But it will create a new layer in Docker...

Reply:
not a problem